frontend/src/components/GithubMirrorModal.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <div>
+    <button
+      class="btn btn-sm btn-outline-secondary w-100-with-icon my-1"
+      rel="nofollow"
+      @click="modal?.show()"
+    >
+      <i class="icon-left fas fa-arrow-up"></i>
+      Mirror on Github
+    </button>
+    <BootstrapModal
+      id="github-mirror-modal"
+      title="Mirror your model on Github"
+      ref="modal"
+      centered
+    >
+      <template #body>
+        <p>
+          This will transform your model into a git repository and archive it on GitHub under a
+          central CoMSES Model Library organization.
```

Comment:
Currently have:

>  This will transform your model into a git repository and archive it on GitHub under a central CoMSES Model Library organization. This is intended to be a read-only archive that allows others to easily view your model's source code and documentation without disrupting the normal CoMSES Model Library workflow.
>
> If you wish to work with the generated repository under your own GitHub account, you can fork it or wait for the upcoming GitHub Sync feature.
>
>Learn more about the GitHub Integration here 


felt like a concise mix of what it does, why, and what to do if you don't like it